Ginger/GingerCoreCommon/Repository/RepositoryFolder.cs

```diff
@@ -696,12 +696,15 @@ public override RepositoryFolderBase AddSubFolder(string folderName)
         /// Save the Repository Item to folder and add it to cache
         /// </summary>
         /// <param name="repositoryItem"></param>
-        public override void AddRepositoryItem(RepositoryItemBase repositoryItem)
+        public override void AddRepositoryItem(RepositoryItemBase repositoryItem, bool DoNotSave = false)
```

Comment:
flag name should be "doNotSave" with "d", sorry for having capital "D" on previous comment